.github/workflows/land-blocking.yml

```diff
@@ -57,11 +74,22 @@ jobs:
           date
           export CTI_OUTPUT_LOG=$(mktemp)
           echo "::set-env name=CTI_OUTPUT_LOG::$CTI_OUTPUT_LOG"
-          ./scripts/cti \
-            --tag ${TEST_TAG} \
-            -E RUST_LOG=debug \
-            --report report.json \
-            --suite land_blocking
+          if [ $TEST_COMPAT -eq 1 ]; then
+            ./scripts/cti \
+              --tag ${PREV_TAG} \
+              --cluster-test-tag ${TEST_TAG} \
+              -E RUST_LOG=debug \
+              -E BATCH_SIZE=15 \
+              -E UPDATE_TO_TAG=${TEST_TAG} \
+              --report report.json \
+              --suite land_blocking_compat
+          else
```

Comment:
@andll I thought we want to run land-blocking on all PRs

Reply:
Yes, so as @rustielin mentioned, we want to run some cluster test on all PRs

By default, we are running land blocking + compat test, unless user explicitly specifies that the commit is a breaking change, in which case we are falling back to regular becnhmark without compatibility.

Reason for this is that we _are_ going to have breaking changes, so we need to account for this in the our land hook